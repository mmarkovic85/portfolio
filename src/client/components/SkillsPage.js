import React from "react";

const SkillsPage = () => (
  <h1>Skills page placeholder</h1>
);

export {
  SkillsPage as default
};
