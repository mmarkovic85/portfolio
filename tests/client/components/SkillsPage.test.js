import React from "react";
import renderer from "react-test-renderer";
import SkillsPage from "../../../src/client/components/SkillsPage";

test("Should render SkillsPage component correctly", () => {
  const rendered = renderer.create(
    <SkillsPage />
  );

  expect(rendered.toJSON()).toMatchSnapshot();
});
