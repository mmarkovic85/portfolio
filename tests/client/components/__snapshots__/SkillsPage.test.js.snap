// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render SkillsPage component correctly 1`] = `
<h1>
  Skills page placeholder
</h1>
`;
